src/clappr-dash-shaka-playback.js

```diff
@@ -99,7 +108,11 @@ class DashShakaPlayback extends HTML5Video {
   }
 
   getProgramDateTime() {
-    return new Date((this.presentationTimeline.getPresentationStartTime() + this.seekRange.start) * 1000)
+    let presentationStartTime = this.presentationTimeline
+      ? this.presentationTimeline.getPresentationStartTime()
```

Comment:
Because in my debug, the getPresentationStartTime() returned null when not available _(i focused to avoid backward compatibility break, since i did not dig deeper on where the functoin was used by yet)_.

Reply:
But i think you are right; actually this is something i also noted with previous code and it worked because `null + this.seekRange.start) * 1000 = 0`. I noted also the sometimes `this.seekRange.start` also equals null